### PR TITLE
Change test runs and coverage locations

### PR DESCRIPTION
**Pull request by anthonydresser**
- Adds unit tests to each of the platforms
- Only publish coverage from darwin

### CODE REVIEW

File: build/azure-pipelines/win32/sql-product-build-win32.yml [hunk 1]
```diff
@@ -108,7 +108,15 @@ steps:
...
-      exec { .\scripts\test-unstable.bat --build --coverage --reporter mocha-junit-reporter }
+      exec { .\scripts\test.bat --build --tfs "Unit Tests" }
+    continueOnError: true
+    condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+    displayName: Run unstable tests
+
+  - powershell: |
+      . build/azure-pipelines/win32/exec.ps1
+      $ErrorActionPreference = "Stop"
+      exec { .\scripts\test-unstable.bat --build --tfs "Unit Tests" }
```
Comment (Charles-Gagnon) [inline build/azure-pipelines/win32/sql-product-build-win32.yml]: Update name to "Unstable Unit Tests"?

File: build/azure-pipelines/win32/sql-product-build-win32.yml [hunk 2]
```diff
@@ -253,6 +261,13 @@ steps:
...
 
+  - task: PublishTestResults@2
+    displayName: 'Publish Test Results test-results.xml'
+    inputs:
+      testResultsFiles: '*.xml'
+      searchFolder: '$(Build.ArtifactStagingDirectory)/test-results'
+    condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+
   - task: PublishTestResults@2
     displayName: 'Publish Test Results test-results.xml'
```
[the remark this answers is not in the document]
Comment (anthonydresser) [reply build/azure-pipelines/win32/sql-product-build-win32.yml]: good catch, i'll just rely on the task above this to publish both xmls

**Comment by kburtram** [comment]
@anthonydresser do we still need this change?  Or should we close out this stale PR?

**Comment by kburtram** [comment]
Closing stale PRs

**Pull request closed without merging**